verify_test.go

```diff
@@ -342,6 +345,97 @@ func TestDistributedClaims(t *testing.T) {
 	}
 }
 
+func TestDistClaimResolver(t *testing.T) {
+	tests := []struct {
```

Comment:
This should just be

```
tests := []resolverTest{
  // ...
}
```

We had the `test.test` in the other ones as a hack :)